C/test/yahdlc_test.cpp

```diff
@@ -3,7 +3,7 @@
 #include <boost/test/unit_test.hpp>
 #include "yahdlc.h"
```

Comment:
Couldn't we instead of making yahdlc_state global simply include "yahdlc.c" into yahdlc_test.cpp?